src/test/groovy/org/prebid/server/functional/tests/TargetingSpec.groovy

```diff
@@ -1121,6 +1126,204 @@ class TargetingSpec extends BaseSpec {
         assert targeting["hb_env"] == HB_ENV_AMP
     }
 
+    def "PBS should throw error when price granularity from original request is empty"() {
```

Comment:
`PBS auction should throw error when price granularity from original request is empty`
In addition you also can add same test for amp endpoint